docs/notebooks/mister_p.ipynb

```diff
@@ -0,0 +1,8615 @@
+{
```

Comment:
<p>nit: warnings is part of the standard library, can you import like this?</p><p> </p><p><code></code><code></p><p>import warnings</p><p> </p><p>import arviz as az</p><p>import bambi as bmb</p><p>import matplotlib.pyplot as plt</p><p>import numpy as np</p><p>import pandas as pd</p><p>import pymc as pm</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>